trantor/utils/AsyncFileLogger.h

```diff
@@ -43,19 +44,19 @@ class AsyncFileLogger : NonCopyable
      * @param msg
      * @param len
      */
-    void output(const char *msg, const uint64_t len);
+    void TRANTOR_EXPORT output(const char *msg, const uint64_t len);
```

Comment:
Please make the location of TRANTOR_EXPORT to be consistent (e.g. place them all before the return type of functions)